### PR TITLE
Feature/post bearing and sensor

### PR DESCRIPTION
**Pull request by Summer0410**
Added 3 endpoints:
1）Add sensor id to a bearing
2)  Post new sensor
3) Posy new bearing

### CODE REVIEW

File: src/app.py [hunk 1]
```diff
@@ -19,5 +19,5 @@
 cors = CORS(app.app)
 
 if __name__ == '__main__':
```
Comment (BenxinNiu) [inline src/app.py]: Debug flag.. and validation flag. Pls lol

File: src/mapper/sensor_summary_mapper.py [hunk 2]
```diff
@@ -2,7 +2,10 @@
 
 
 def to_sensor_list(sensor_id_list):
-    return list(map(lambda id: to_sensor(id), sensor_id_list))
+    sensor_id_list = list(map(lambda id: to_sensor(id), sensor_id_list))
+    for sensor_id in sensor_id_list:
+        sensor_id = str(sensor_id)
+    return sensor_id_list
```
Comment (BenxinNiu) [inline src/mapper/sensor_summary_mapper.py:6]: the naming is a bit confusing... This line `list(map(lambda id: to_sensor(id), sensor_id_list))` returns a list of dictionary. but the name suggests it returns a list of `sensor ids`. Also, the intention of for loop is unclear.  I might have missed something. Pls clarify

File: src/mapper/sensor_summary_mapper.py [hunk 3]
```diff
@@ -13,3 +16,10 @@ def to_sensor(sensor_id):
         "tags": sensor.sensor_tags,
         "installTime": sensor.sensor_install_time
     }
+
+def to_sensor_detail(sensor):
+    return {
+        "id": str(sensor.id),
+        "sensor_status": sensor.sensor_status,
+        "tags": sensor.sensor_tags
+    }
```
Comment (BenxinNiu) [inline src/mapper/sensor_summary_mapper.py:23]: Not sure what is this mapping for? it looks like this method is used in sensor `controller`, but I don't see any objects in the swagger that has `sensor_status` field.

File: src/swagger.yaml [hunk 4]
```diff
@@ -241,6 +240,28 @@ paths:
...
+      parameters:
+      - in: "body"
+        name: "body"
+        description: "Name of the orgnization to which the bearing belongs to"
+        required: true
+        schema:
+          $ref: "#/definitions/PostSensor"
+      responses:
+        405:
+          description: "Invalid input"
```
Comment (BenxinNiu) [inline src/swagger.yaml:301]: maybe it's still work in progress, but there is no `200` response?

**Review by BenxinNiu — CHANGES_REQUESTED** [review]

File: src/mapper/sensor_summary_mapper.py (hunk 2 above)
Comment (Summer0410) [reply src/mapper/sensor_summary_mapper.py:6]: This was left behind when I was trying to fix serialization  problem , got the lines removed.

File: src/swagger.yaml (hunk 4 above)
Comment (Summer0410) [reply src/swagger.yaml:301]: Fixed

File: src/mapper/sensor_summary_mapper.py (hunk 3 above)
Comment (Summer0410) [reply src/mapper/sensor_summary_mapper.py:23]: Fixed, this is what is return when a new sensor is posted.

**Review by BenxinNiu — APPROVED** [review]

**Pull request merged**